docs/CreateEdit.md

```diff
@@ -486,11 +488,11 @@ export const PostCreate = (props) => (
 );
 ```
 
-**Tip**: You can include properties in the form `defaultValue` that are not listed as input components, like the `created_at` property in the previous example.
+**Tip**: You can include properties in the form `initialValues` that are not listed as input components, like the `created_at` property in the previous example.
 
 ### Per Input Default Value
 
-Alternatively, you can specify a `defaultValue` prop directly in `<Input>` components. Just like for form-level default values, an input-level default value can be a scalar, or a function returning a scalar.  React-admin will merge the input default values with the form default value (input > form):
+Alternatively, you can specify a `defaultValue` prop directly in `<Input>` components. Default value can be a scalar, or a function returning a scalar.  React-admin will merge the input default values with the form default value (input > form):
```

Comment:
Note that both `defaultValue` and `initialValue` are supported: https://final-form.org/docs/react-final-form/types/FieldProps#defaultvalue & https://final-form.org/docs/react-final-form/types/FieldProps#initialvalue.

Probably worth to mention that users should read final-form documentation

Reply:
initialValue on input does not work either in SImpleForm or Tabbed Form